### DISCUSSION TITLE
No sound.

### DISCUSSION THREAD

**Discussion by ynnebbenny** [probonopd/MiniDexed#276] [Q&A] [answered]
Your suggestions? .....I have recently assembled a minidexed and the display and button are working, but I dont have any sound.
I have tried editing minidexed.ini by changing the # for the various sound options but still don't get any sound. At minimal I should expect sound via the inbuild Pi sound output, but I get nothing.
I'm suspecting its probably the Launchkey mini that I am using. How do I verify if it is imputing? Is there something further I need to edit in the minidexed.ini to get the Launchkey to trigger a sound?

**Comment by Banana71** [comment]
Someone here has had the same issues:

https://github.com/probonopd/MiniDexed/discussions/258#discussion-4100469

**Reply by probonopd** [reply]
Please attach to a HDMI screen and watch the output. Does your MIDI keyboard work on a PC without special drivers?

**Comment by ynnebbenny** [comment, marked as answer]
In the end I usd a qwerty keyboard and it works, so that confirms that the Launchkey is the issue. Also I connected to a monitor and could see that it didnt accept the launchkey either.
I can confirm that a PiFi DAC+ V2 works with MiniDexed
